src/test/resources/application.properties

```diff
@@ -6,13 +6,11 @@ management.endpoints.web.path-mapping.health=/healthcheck
 uk.gov.companieshouse.documentsigningapi.signpdf=/document-signing/sign-pdf
 
 # Environment variables
-environment.keystore.type=${KEYSTORE_TYPE}
-environment.keystore.path=${KEYSTORE_PATH}
-environment.keystore.password=${KEYSTORE_PASSWORD}
-environment.certificate.alias=${CERTIFICATE_ALIAS}
```

Comment:
Curious to know why the variables are changed to hardcoded values here?

Reply:
The itest env doesn't pick up the values created after upgrade. Only works when hardcoded in.